app/main.py

```diff
@@ -1,3 +1,75 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(km = self.km + other)
+        elif isinstance(other, Distance):
+            return Distance(km = self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, (int, float)):
+            self.km += other
+        elif isinstance(other, Distance):
+            self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(km = self.km * other)
```

Comment:
The `__mul__` method currently only handles multiplication with `int` or `float`. Consider adding support for multiplication with another `Distance` object, or raise a `TypeError` if the operation is not supported.